release.md

```diff
@@ -0,0 +1,63 @@
+
+>_Note that not all versioned releases will be built and distributed. Drivers distribution is indeed implemented and maintained only to satisfy Falco's needs._
+
+The libs release process is currently under development and limited to the versioning process only. The [libs versioning](#Libs-versioning) process happens in this repo.
+
+
+## Release procedure
+
+Regardless if it is a driver or a libs release when initiating a new release, we do the following process:
+
+1. We decide together (usually in the #falco channel in [slack](https://kubernetes.slack.com/messages/falco)) if the source code is a good shape and if it's the case to be released
+2. We double-check if the versioning rules have been respected (see sections below), then we pick the next version number to tag (i.e., a _git tag_)
+3. A person with repository rights creates a [new release using the GitHub UI](https://github.com/falcosecurity/libs/releases/new) (a git tag will be automatically created)
+
+> _At the time of writing, no other steps are needed since the whole release process is still in development. This document will be updated once the definitive process is fully implemented._
+
+
+## Drivers versioning
+
+The *driver version number* represents the build version of kernel-space drivers (i.e., the kernel module, the eBPF probe, and possibly any other kernel-space drivers).
+
+**Requirements**
+
+- The version MUST be a [SemVer 2.0](https://semver.org/spec/v2.0.0.html) string.
+
+- Since our driver APIs are assumed to be stable, the major version number MUST be equal to or greater than `1`.
+
+- The version string MUST be suffixed with `+driver` to distinguish it from libs version numbers.
+
+- To pick a new driver version number:
+
+    1. The [API for user/kernel boundary](https://github.com/falcosecurity/libs/blob/master/proposals/20210818-driver-semver.md) is versioned in the source code. You MUST ensure the two files below have been updated by following the rules described in [driver/README.VERSION.md](https://github.com/falcosecurity/libs/blob/master/driver/README.VERSION.md):
+        - [driver/API_VERSION](https://github.com/falcosecurity/libs/blob/master/driver/API_VERSION)
+        - [driver/SCHEMA_VERSION](https://github.com/falcosecurity/libs/blob/master/driver/SCHEMA_VERSION)
+    
+        If not, open a PR to fix them.
+
+    2. Compute the *driver version number* by strictly following the [Versioning Schema](https://github.com/falcosecurity/libs/blob/master/proposals/20220203-versioning-schema-amendment.md#drivers-artifacts) proposal's rules.
+
+> _Note that `API_VERSION` and `SCHEMA_VERSION` are only used internally. On the other hand, only the **driver version number** will be used **to tag a new release**._
+
+
+
```

Comment:
Ultra-nit, leave a single empty line here, it doesn't make much difference to markdown so up to you if you want to remove them.